src/app/components/template-button/template-button.component.ts

```diff
@@ -8,6 +8,8 @@ import { Component, Input } from '@angular/core';
 export class TemplateButtonComponent {
     @Input() buttonText: string | undefined;
     @Input() buttonIcon: string | undefined;
+    @Output() buttonAction: EventEmitter<any> = new EventEmitter();
+    @Output() dropAction: EventEmitter<any> = new EventEmitter();
```

Comment:
Der Typ kann wie gesagt bleiben, wenn du magst, aber das `any` sollte hier durch `DragEvent` ersetzt werden. 

```suggestion
    @Output() dropAction = new EventEmitter<DragEvent>();
```